src/instruments/src/EFB/index.tsx

```diff
@@ -49,6 +53,8 @@ const EFBLoad = () => {
         }, 6000);
     }
 
+    useEffect(() => () => setSessionId(''), []);
```

Comment:
Won't this always clear the `A32NX_SENTRY_SESSION_ID` persistent property when the EFB intiializes?

As the `generatedSessionID` below will only be set once on a JS load, before the EFB renders.

Reply:
> Won't this always clear the `A32NX_SENTRY_SESSION_ID` persistent property when the EFB intiializes?
> 
> 
> 
> As the `generatedSessionID` below will only be set once on a JS load, before the EFB renders.

Wouldn't the cleanup function for the useffect run only when the component gets unmounted?

Reply:
Ah, you're right, I missed the double-arrow.